### DISCUSSION TITLE
IPv6 for service networks

### DISCUSSION THREAD

**Discussion by Bilge** [coollabsio/coolify#4334] [Feature Requests]
Service stacks are deployed to their own individual networks, and even with the ability to edit the raw Docker Compose file, the network configuration is not exposed at all. This prohibits the ability to add IPv6 subnets to the stack. Either expose the raw network configuration, or add first-class support for IPv6 in the UI, including a checkbox to enable IPv6 and an editable list of subnets to delegate.

**Comment by felix-schindler** [comment]
There are also some bugs like #3436 because Coolify doesn't create the docker networks with IPv6 enabled. This should be the default for all networks.
